pkg/pipelines/bootstrap.go

```diff
@@ -82,14 +83,23 @@ func Bootstrap(o *BootstrapParameters) error {
 	}
 
 	if o.GithubToken != "" {
-		githubAuth, err := createOpaqueSecret(meta.NamespacedName(namespaces["cicd"], "github-auth"), o.GithubToken)
+		githubAuth, err := createOpaqueSecret(meta.NamespacedName(namespaces["cicd"], "github-auth"), o.GithubToken, "token")
 		if err != nil {
 			return fmt.Errorf("failed to generate path to file: %w", err)
 		}
 
 		outputs = append(outputs, githubAuth)
 	}
 
+	if o.GithubHookSecret != "" {
+		githubSecret, err := createOpaqueSecret(meta.NamespacedName(namespaces["cicd"], "github-webhook-secret"), o.GithubHookSecret, "webhook-secret-key")
+		if err != nil {
```

Comment:
Please use const for "github-webhook-secret" and "webhook-secret-key" and pass them to create interceptor func.